teuthology/openstack/__init__.py

```diff
@@ -830,17 +830,21 @@ def create_security_group(self):
             return
         except subprocess.CalledProcessError:
             pass
-        # TODO(loic): this leaves the teuthology vm very exposed
-        # it would be better to be very liberal for 192.168.0.0/16
-        # and 172.16.0.0/12 and 10.0.0.0/8 and only allow 80/8081/22
-        # for the rest.
+        # TODO(jan): is there a way teuthology can exist but teuthology worker
+        # doesn't? if so the detection logic needs an update.
```

Comment:
The try-catch block above checks for presence of the ```teuthology``` security group and skips the rule creation if it is present. Was wondering about the possibility of ```teuthology``` being present but not ```teuthology-worker```.

But this is probably not really an issue.